_docs-2/administration/upgrading.md

```diff
@@ -38,7 +57,70 @@ will then need to transfer the contents of the current individual files to this
 **Warning**: Upgrading a previously encrypted instance is not supported as the implementation
 and properties have changed.
 
-## Upgrading from 1.8/9 to 2.0
+## Upgrading from 1.9/1.10/2.0 to 2.1
+
+Note: the process for upgrading to 2.1 assumes that you have already upgraded to 1.10 or 2.0.  Upgrading from a 
+version prior to 1.10 may work, but is unsupported and untested.  If you are upgrading from 1.10 make sure
```

Comment:
Lots of things "may" work. I don't think we need to speculate on those scenarios. Also, we know that *most* versions prior to 1.10 definitely won't work, so this language is really sub-optimal. We should focus on the supported upgrade paths, and direct people to the mailing list for help with anomalous scenarios.

Reply:
This was trying to hedge that it seems likely that because the changes between 1.9.3 and 1.10 are small it seems likely that upgrading from 1.9.3 to 2.10 would probability work - but I have not tried it.